android/fastlane/Fastfile

```diff
@@ -23,59 +23,98 @@ platform :android do
 
   JS_PACKAGE = load_json(json_path: "../package.json")
 
-  desc "Build a Staging APK and AAB"
-  lane :staging do
+  desc "GPS: Build a Staging AAB"
+  lane :staging_gps do
     gradle(
-      task: "clean assembleStagingDebug",
+      task: "clean assemble",
+      build_type: "gpsStaging",
       print_command: false,
     )
   end 
 
-  desc "Build a Release APK and AAB"
-  lane :release do
+  desc "GPS: Build a Release AAB"
+  lane :release_gps do
 
-    increment_version_code(
-      gradle_file_path: "../android/app/build.gradle",
-    )
+    build_number =  ENV["GITHUB_RUN_NUMBER"]
```

Comment:
I dig this! i would double check that its not conflicting with some of the work that is being done here: https://github.com/Path-Check/covid-safe-paths/pull/873

let me know if you wanna chat through it some more, but Id love to get the buildnumbers for the ios side incrementing and independent of the version number as well, its a bit odd to have the version number change along with the build number and means that we have many many versions.

Reply:
I like @tstirrat's approach in #873 - we'd read the new version name in `package.json. The `versionCode` on android and the `build number` on iOS should be independent, and increment automatically on each workflow run. What do you think?

Reply:
tbh, i think we could be manually upping the build numbers, that said, having some automation is nice and if tim's strategy works then it works!

Reply:
sounds good! this PR won't address versionName; in the interest of time we can keep the versionCode incrementation logic here and we can all discuss the versionName piece or revisit this question as part of #873

Reply:
Yeah, version codes can be whaterver as long as they always increase. Seems good.